### ISSUE TITLE
Collect email addresses?

### ISSUE THREAD

**Issue by Kurt124** [al-caughey/Google-Meet-Attendance#123]
Is there any way your attendance program could collect email addresses? Students that are not supposed to be in the Meet are changing their names to people in the Meet. The teacher sees the name and lets them in...but they are not supposed to be there.  If we collected the email addresses maybe we could track those people down?

Thanks,
Kurt

**Comment by Eagle-Man** [comment]
Nope.  Google doesn't make that information available to the public for security/privacy reasons.  If you want that info, then your organization will have to pay for an Enterprise/Education+/Teacher and Learning Upgrade, and use the attendance report sent by Google.
